rpc-server/src/config.rs

```diff
@@ -64,6 +64,24 @@ pub struct Opts {
     /// Default value is 300_000_000_000_000
     #[clap(long, env, default_value = "300000000000000")]
     pub max_gas_burnt: near_primitives_core::types::Gas,
+
+    /// Max available memory for `block_cache` and `contract_code_cache` in bytes
```

Comment:
That's the opposite of good UX, forcing people to provide raw bytes when dealing with gigabytes. Please accept gigabytes here and convert them into bytes later in the code.

Do this for every place where you were going to ask people for bytes.